Sources/DotNetGraph/Compiler/DotCompiler.cs

```diff
@@ -192,7 +193,7 @@ private void CompileAttributes(StringBuilder builder, ReadOnlyCollection<IDotAtt
                 }
                 else if (attribute is DotNodeHeightAttribute nodeHeightAttribute)
                 {
-                    attributeValues.Add($"height={nodeHeightAttribute.Value:F2}");
+                    attributeValues.Add(string.Format(CultureInfo.InvariantCulture, "height={0:F2}", (object) nodeHeightAttribute.Value));
```

Comment:
The `(object)` cast is not required here, you can remove it.

Reply:
Corrected 🙂